README.md

```diff
@@ -142,6 +142,8 @@ with s.time():
   pass
 ```
 
+**Summary support is incomplete**. The python client doesn't store or expose quantile information at this time.
```

Comment:
That's a bit strong, this is a complete client library per the spec. I'd remove the first sentence.